mappings/net/minecraft/world/gen/chunk/ChunkNoiseSampler.mapping

```diff
@@ -0,0 +1,152 @@
+		ARG 1 x
+		ARG 2 z
+	METHOD m_eyivlffz createMultiNoisePoint (Lnet/minecraft/unmapped/C_wgwzecpu;II)Lnet/minecraft/unmapped/C_unqwyjvr$C_ydritevk;
+		ARG 0 columnSampler
+		ARG 1 x
+		ARG 2 z
+	METHOD m_fbsgezeh getTerrainNoisePoint (II)Lnet/minecraft/unmapped/C_bnmpwohp;
+		ARG 1 x
+		ARG 2 z
+	METHOD m_hqlkbjrc sampleNoiseX0 ()V
+	METHOD m_jaocstik sampleNoiseX (D)V
+		ARG 1 deltaX
+	METHOD m_kixunegf getErosion (II)D
+		ARG 1 x
+		ARG 2 z
+	METHOD m_nxzvbtxe getNoiseX (II)D
+		ARG 1 x
+		ARG 2 z
+	METHOD m_ogdcmgqp sampleNoiseZ (D)V
+		ARG 1 deltaZ
+	METHOD m_ovecuktv getNoiseZ (II)D
+		ARG 1 x
+		ARG 2 z
+	METHOD m_pzogpiqn sampleNoiseX1 (I)V
+		ARG 1 x
+	METHOD m_qigvbhjb sampleNoiseCorners (II)V
+		ARG 1 noiseY
+		ARG 2 noiseZ
+	METHOD m_rbjtrunh (Lnet/minecraft/unmapped/C_wgwzecpu;J)Lnet/minecraft/unmapped/C_bnmpwohp;
+		ARG 1 pos
+	METHOD m_rhmdxspk swapBuffers ()V
+	METHOD m_slafubek getTerrainNoisePoint (Lnet/minecraft/unmapped/C_wgwzecpu;II)Lnet/minecraft/unmapped/C_bnmpwohp;
+		ARG 1 columnSampler
+		ARG 2 x
+		ARG 3 z
+	METHOD m_vibcskmo (III)Lnet/minecraft/unmapped/C_pmeibtxt;
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD m_wfsrcqqn sampleNoiseY (D)V
+		ARG 1 deltaY
+	METHOD m_xyanyqky createNoiseInterpolator (Lnet/minecraft/unmapped/C_unqwyjvr$C_sdkmrqke;)Lnet/minecraft/unmapped/C_unqwyjvr$C_utnexdzf;
+		ARG 1 sampler
+	METHOD m_ynijfsfr getAquiferSampler ()Lnet/minecraft/unmapped/C_cxzdhgpj;
+	CLASS C_dmlpsnok ValueSampler
+		METHOD m_yqvwvobh sample ()D
+	CLASS C_poowcsja BlockStateSampler
+		METHOD m_mdcbjuxu getBlockState (III)Lnet/minecraft/unmapped/C_pmeibtxt;
+			ARG 1 x
+			ARG 2 y
+			ARG 3 z
+	CLASS C_sdkmrqke NoiseSampler
+		METHOD m_kueojymf sample (III)D
+			ARG 1 x
+			ARG 2 y
+			ARG 3 z
+	CLASS C_utnexdzf NoiseInterpolator
+		FIELD f_aewbqvwn x0y1z1 D
+		FIELD f_bbhngjkr x0y1z0 D
+		FIELD f_gibokijz x1y1z0 D
+		FIELD f_htbpftbb z1 D
+		FIELD f_icmssybp sampler Lnet/minecraft/unmapped/C_unqwyjvr$C_sdkmrqke;
+		FIELD f_islmmpce z0 D
+		FIELD f_keujnqxw x0z0 D
+		FIELD f_lztvxhuc x1y0z0 D
+		FIELD f_mctqespr result D
+		FIELD f_mejpfspb noiseBufferX0 [[D
+		FIELD f_mlcswjik x0z1 D
+		FIELD f_ntforull x0y0z0 D
+		FIELD f_pdvekohd x1y1z1 D
+		FIELD f_puyvfpah x0y0z1 D
+		FIELD f_sfmejueb noiseBufferX1 [[D
+		FIELD f_sppcnxzx x1z1 D
+		FIELD f_ygiitukj x1y0z1 D
+		FIELD f_zjliborh x1z0 D
+		METHOD <init> (Lnet/minecraft/unmapped/C_unqwyjvr;Lnet/minecraft/unmapped/C_unqwyjvr$C_sdkmrqke;)V
+			ARG 2 sampler
+		METHOD m_bzduzvlg sampleNoiseZ (D)V
+			ARG 1 deltaZ
+		METHOD m_cohgrnoy swapBuffers ()V
+		METHOD m_ffcniepn sampleNoiseY (D)V
+			ARG 1 deltaY
+		METHOD m_gdohomuy sampleNoiseX0 ()V
+		METHOD m_jvrpqhak sampleNoiseCorners (II)V
+			ARG 1 noiseY
+			ARG 2 noiseZ
+		METHOD m_kcbykdlu sampleNoiseX (D)V
+			ARG 1 deltaX
+		METHOD m_lwxaztkx createBuffer (II)[[D
+			ARG 1 noiseY
+			ARG 2 noiseZ
+		METHOD m_wjebstow sampleNoise ([[DI)V
+			ARG 1 buffer
+			ARG 2 noiseX
+		METHOD m_wpfrryvm sampleNoiseX1 (I)V
```

Comment:
These don't seem right?

Reply:
What do you mean exactly? Also, these aren't new mappings, they're also from Yarn.
Just changed `sampleStartNoise` to `sampleNoiseX0` and `sampleEndNoise` to `sampleNoiseX1`.